core/org.openjdk.jmc.flightrecorder.rules/src/main/java/org/openjdk/jmc/flightrecorder/rules/report/JfrRulesReport.java

```diff
@@ -329,8 +329,9 @@ public int compare(Entry<IRule, ?> o1, Entry<IRule, ?> o2) {
 								Element itemNode = parent.getOwnerDocument().createElement("item"); //$NON-NLS-1$
 								itemsNode.appendChild(itemNode);
 								for (IMemberAccessor<?, IItem> a : accessors) {
-									itemNode.appendChild(createValueNode(parent.getOwnerDocument(), "value", //$NON-NLS-1$
-											toString(a.getMember(item))));
+									if (a != null)
```

Comment:
Added this null check here to avoid further NPEs due to null accessors.

Reply:
This shouldn't happen normally. Should there be some logging so that we can discover if it does?